Soccar [UI]/src/nl/soccar/ui/fx/GameCanvasFx.java

```diff
@@ -5,13 +5,19 @@
 import javafx.scene.canvas.Canvas;
 import javafx.scene.canvas.GraphicsContext;
 import javafx.util.Duration;
+import nl.soccar.library.Event;
 import nl.soccar.library.Game;
+import nl.soccar.library.Notification;
+import nl.soccar.library.enumeration.GameStatus;
+import nl.soccar.physics.PhysicsConstants;
+import nl.soccar.ui.DisplayConstants;
 import nl.soccar.ui.drawable.Drawable;
 import nl.soccar.ui.drawable.GameCanvas;
+import nl.soccar.ui.fx.drawable.NotificationUiFx;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.ui.fx.drawable.NotificationUiFx'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)